csg/share/scripts/inverse/inverse.sh

```diff
@@ -100,14 +100,18 @@ while [[ ${1#-} != $1 ]]; do
 done
 ### end parsing options
 
-#old style, inform user
+# old style, inform user
 [[ -z ${CSGXMLFILE} ]] && die "Please add your setting xml file behind the --options option (like for all other votca programs) !"
 
 [[ $1 = "clean" ]] && { csg_inverse_clean "$waittime"; exit $?; }
 
 enable_logging
 [[ -n $CSGDEBUG ]] && set -x
 check_for_obsolete_xml_options
+check_xml_output=$(do_external check csg_xml "$CSGXMLFILE" "$VOTCA_CSG_DEFAULTS")
+if [[ -n $check_xml_output ]]; then
+  msg --color blue --to-stderr "WARNING: $check_xml_output"
```

Comment:
```suggestion
  msg --color blue --to-stderr "WARNING: $check_xml_output"
  [[ ${CI} = true ]] && die "Clean-up with xml option above"
```
so on the CI is still fails.